docs/content/cli-commands/npm-dedupe.md

```diff
@@ -60,6 +60,8 @@ Modules
 Note that this operation transforms the dependency tree, but will never
 result in new modules being installed.
 
+Using `npm find-dupes` will run the command in dryRun mode.
+
```

Comment:
oh if `npm find-dupes` is now a top-level command it'd be nice to also add docs for it and a link below in the **See Also** section of this file 👇

Reply:
I'm not sure about creating a new docs file for this command since it is essentially the same as `dedupe` only on dryRun mode. What do you think about just pointing the `find-dupes` docs to the `dedupe` docs?

Reply:
right! sounds good 👍